### DISCUSSION TITLE
关于段落行间距的问题

### DISCUSSION THREAD

**Discussion by wayne-sjtu** [sjtug/SJTUThesis#862] [提问] [answered]
请问想要修改一下正文的行间距是怎么修改，现在的行间距看起来太小了

**Comment by LogCreative** [comment, marked as answer]
行间距是通过本行 `linespread` 参数设定的，数值是由论文规范规定的。

https://github.com/sjtug/SJTUThesis/blob/700093d999dc6fe212aa280d04aee6c120470f3b/texmf/tex/latex/sjtuthesis/sjtuthesis.cls#L185

如果需要更改，在导言区使用
```latex
\linespread{1.4}\selectfont
```
或
```latex
\ctexset{linespread=1.4}
```

引自 [CTeX 宏集手册](https://mirrors.sjtug.sjtu.edu.cn/CTAN/language/chinese/ctex/ctex.pdf)：
> 接受一个浮点数值，设置行距倍数。
> 对标准文档类初始值为 1.3，即 1.3 倍行距。此时，相邻两行的基线(\baselineskip)距离为 1.3 × 1.2 = 1.56 倍字体高度。

**Reply by LogCreative** [reply]
修订：使用 `\linespread` 命令时，其后应当添加 `\selectfont`，否则该命令可能不起作用。
